gotrue/types.py

```diff
@@ -41,6 +41,10 @@
     "zoom",
 ]
 
+EmailOtpType = Literal[
+    "signup", "invite", "magiclink", "recovery", "email_change", "email"
```

Comment:
Not in scope for this PR but We should probably deprecate these at some point right: https://github.com/supabase/supabase/pull/14091